src/test/resources/ovs/v2/ovsTimeStamps/TimeStampsSample.json

```diff
@@ -16,7 +16,7 @@
     },
     "nmftaCode": "MMCU"
   },
-  "publisherRole": "DDS",
+  "publisherRole": "ATH",
```

Comment:
```suggestion
  "publisherRole": "CA",
```

Reply:
Any reason for changing it to CA instead of ATH?

Reply:
This is because it makes this PR exactly the same as my #79 PR. My PR succeed all tests - I wanted to investigate why there is a difference in number of tests failing?!? You can change it back to ATH or keep CA...